tests/clusters/test_cluster.py

```diff
@@ -0,0 +1,64 @@
+import importlib
+from unittest.mock import patch
+
+from snuba import settings
+from snuba.clickhouse.native import ClickhousePool
+from snuba.clusters import cluster
+from snuba.datasets.storages import StorageKey
+from snuba.datasets.storages.factory import get_storage
+
+
+class TestClusters:
+    def setup_class(self) -> None:
+        settings.CLUSTERS = [
+            {
+                "host": "host_1",
+                "port": 9000,
+                "http_port": 8123,
+                "storage_sets": {"events", "outcomes", "querylog", "sessions"},
+                "distributed": False,
+            },
+            {
+                "host": "host_2",
+                "port": 9000,
+                "http_port": 8123,
+                "storage_sets": {"transactions"},
+                "distributed": True,
+                "cluster_name": "test_distributed_cluster",
+            },
+        ]
+        importlib.reload(cluster)
+
+    def teardown_class(self) -> None:
+        importlib.reload(settings)
+        importlib.reload(cluster)
```

Comment:
This is really annoying, but I don't have an alternative idea for how to restore settings / cluster modules here.